cmd/ris-mirror/rismirror/rismirror.go

```diff
@@ -35,6 +36,41 @@ func (rism *RISMirror) AddTarget(rtrName string, address net.IP, vrfRD uint64, s
 	r.addVRF(vrfRD, sources)
 }
 
+func (rism *RISMirror) RemoveTarget(rtrName string, vrfRD uint64) {
+	rism.routersMu.Lock()
+	defer rism.routersMu.Unlock()
+
+	if _, exists := rism.routers[rtrName]; !exists {
+		log.Errorf("Router %v does not exist, cannot remove cleanly", rtrName)
+		delete(rism.routers, rtrName)
+		return
+	}
+	r := rism.routers[rtrName].(*Router)
+	// remove chosen vrf
```

Comment:
I think this comment is pretty redundant with the next line.

Reply:
Done.